README.md

```diff
@@ -1,180 +1,180 @@
+## BaseCombinedDrawable
 
-##Api介绍
+A combined drawable can piece together two images and combine them into a new element.
+The positions of the two images can be flexibly combined, which can basically meet the customization of all images, and a variety of graphics .
 
-```java
+API
 
-     /**
-     * 设置drawable two 左上角相对于 drawable one左上角 的相对偏移位置
-     * 偏移以drawable one 的左上角为起始点
-     * drawable one 会优先放在最前面进行绘制(如果两个drawable的相对偏移值为0则效果如同FrameLayout)
+```groovy
+    /**
+     * Set the relative offset position of the upper left corner of drawable two relative to the upper left corner of
+     * drawable one.
      *
-     * @param relatedX x轴的相对偏移
-     * @param relatedY y轴的相对偏移值
+     * @param relatedX relative offset of x-axis
+     * @param relatedY relative offset of y-axis
      */
-     public void setRelatedPosition(int relatedX, int relatedY)
-    
-    
+	 public void setRelatedPosition( int relatedX, int relatedY)     
+      
     /**
-     * 设置组合后的新的drawable的四个Padding值
+     * Method to set padding on four sides.
      *
-     * @param paddingLeft   左边填充距离
-     * @param paddingTop    上边填充距离
-     * @param paddingRight  右边填充距离
-     * @param paddingBottom 下边填充距离
+     * @param paddingLeft   Left padding
+     * @param paddingTop    Top padding
+     * @param paddingRight  Right padding
+     * @param paddingBottom Bottom padding
      */
-     public void setPadding(int paddingLeft, int paddingTop, int paddingRight, int paddingBottom)
-    
+	 public void setPadding ( int paddingLeft, int paddingTop, int paddingRight, int paddingBottom)
 ```
 
-##代码示例
-
-```java
-
-        BaseCombinedDrawable baseCombinedDrawable = new BaseCombinedDrawable(drawableLeft, drawableRight);
-        baseCombinedDrawable.setRelatedPosition(drawableLeft.getIntrinsicWidth() + drawablePadding, 0);
-        combine.setImageDrawable(baseCombinedDrawable);
-
+Sample code
+
+```groovy
+	    BaseCombinedDrawable baseCombinedDrawable = new BaseCombinedDrawable(drawableLeft, drawableRight);
+        baseCombinedDrawable.setRelatedPosition(drawableLeft.getWidth() + drawablePadding, 0);
+        Image combinedImg = (Image) findComponentById(ResourceTable.Id_combinedImage);
+        combinedImg.addDrawTask(new Component.DrawTask() {
+            @Override
+            public void onDraw(Component component, Canvas canvas) {
+                baseCombinedDrawable.drawToCanvas(canvas);
+            }
+        });
+        combinedImg.invalidate();
 ```
 
+## Future work
 
-##效果图:
-<img src="https://github.com/wuseal/TextWithImageDrawable/blob/master/demo.png?raw=true" alt="alt text" >
- 
+Since there is no alternate api for setColorFilter and getOpacity in HMOS platform, so those functions is currently not supported. 
+This library draws the image and text directly on the provided canvas, so we cannot use TextWithImageDrawable as an input to BaseCombinedDrawable and width and height of the image should be managed properly
```

Comment:
"This library draws the image and text directly on the provided canvas, so we cannot use TextWithImageDrawable as an input to BaseCombinedDrawable. Also, width and height of the image should be managed accordingly" -- Add this at the end of Usage section. (Remove it from Future work section)